Editor/Resources/EditorWindow/GameLiftPlugin.uxml

```diff
@@ -0,0 +1,33 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" engine="UnityEngine.UIElements" editor="UnityEditor.UIElements" noNamespaceSchemaLocation="../../../../../UIElementsSchema/UIElements.xsd" editor-extension-mode="False">
+    <Style src="./GameLiftPlugin.uss" />
+    <ui:VisualElement name="Main" class="main">
+        <ui:VisualElement class="main__menu">
+            <ui:Button text="Amazon GameLift" name="Landing" class="TabButton" />
+            <ui:VisualElement class="Divider" />
+            <ui:Button text="AWS Account Credentials" name="Credentials" class="TabButton" />
+            <ui:VisualElement class="Divider" />
+            <ui:Button text="GameLift Anywhere" name="Anywhere" class="TabButton" />
+            <ui:VisualElement class="Divider" />
+            <ui:Button text="Managed EC2" name="EC2" class="TabButton" />
+            <ui:VisualElement class="Divider" />
+            <ui:Button text="Help &amp; Documentation" name="Help" class="TabButton" />
+        </ui:VisualElement>
+        <ui:ScrollView name="Content" horizontal-scroller-visibility="Hidden" class="main__content">
+            <ui:VisualElement name="LandingTab" class="TabContent">
+                <ui:Label text="Landing" />
+            </ui:VisualElement>
+            <ui:VisualElement name="CredentialsTab" class="TabContent">
+                <ui:Label text="Credentials" />
+            </ui:VisualElement>
+            <ui:VisualElement name="AnywhereTab" class="TabContent">
+                <ui:Label text="Anywhere" />
+            </ui:VisualElement>
+            <ui:VisualElement name="EC2Tab" class="TabContent">
+                <ui:Label text="Managed EC2" />
```

Comment:
Should these strings be pulled from the TextProvider? Why are these hard coded?

Reply:
These have default strings so that within the UI Builder tool in unity the elements have some form of text as the tool only renders the uxml and uss files.

As long as we are using text provider to write the strings in `CreateGUI` then these will not show on first paint

Reply:
Ah got it, so the difference is for development vs actually rendering as a plugin, in which case we'd pull from the TextProvider?

Reply:
Yes thats correct